lib/cookie.js

```diff
@@ -980,15 +1006,36 @@ function CookieJar(store, options) {
   if (options.looseMode != null) {
     this.enableLooseMode = options.looseMode;
   }
+  if (options.prefixSecurity != null) {
+    const normalizedPrefixSecurity = options.prefixSecurity.toLowerCase();
+    switch (normalizedPrefixSecurity) {
+      /* The three supported options */
+      case PrefixSecurityEnum.STRICT:
+      case PrefixSecurityEnum.SILENT:
+      case PrefixSecurityEnum.DISABLED:
```

Comment:
Why do we need the multi-line CASE here? Future proofing? Just wondering if this might be better expressed for now as a simple setter.

Reply:
Because we are only accepting those 3 options, right? If it's not one of the 3 options, then we opt for the default (silent).